pkg/qe-tests/cypress/integration/models/plan.ts

```diff
@@ -573,7 +575,7 @@ export class Plan {
       this.filterVm(duplicatePlanData);
       this.selectVm(duplicatePlanData);
       this.unSelectVm(vmListRemove);
-      this.networkMappingStep(duplicatePlanData);
```

Comment:
@ibragins sorry for the layman question, why do we skip the netwok step?

Reply:
Cause there is no change expected. Also if I will expect any change in network mapping - I will need to control some how if this a new plan creation or editing/duplicating it which will cause serious refactoring. I was asked not to put too much effort in this.